src/main/java/org/zone/utils/Messages.java

```diff
@@ -253,7 +255,7 @@ public static Component getZoneFlagLeavingMessageSet(Component message) {
         return Component
                 .text("Leaving message is now: ")
                 .color(NamedTextColor.AQUA)
-                .append(message);
+                .append(message.color(NamedTextColor.GOLD));
```

Comment:
This should not be forcing a component to be gold. The message should be set to gold before passed in

Reply:
K

Reply:
can't be done bruh

Reply:
The only place this method is used is on the LeavingSet command. While it comes in as a Component, it is meant to be this as the player can set the colour and the message they are setting in its entirity. So remove this change

Reply:
Uhm m

Reply:
Still waiting for removal

Reply:
already done